react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -230,7 +230,7 @@ const defaultStyles = theme => {
                 marginRight: '4px'
             },
 
-            '&:not(.top-indicators) > *:last-child': {
+            '&:not(.top-indicators) > span:last-child': {
```

Comment:
Why was this changed?

Reply:
Because this would've applied on the last status indicator (div) and it would've looked funny. 
![image](https://user-images.githubusercontent.com/20574669/146551764-a79ca7b1-b34d-410d-9b2f-18712542b504.png)

I saw a lot of logic in the css selectors, are we avoiding classnames by choice?